cpp/tests/io/comp/decomp_test.cpp

```diff
@@ -18,10 +18,13 @@
 
 #include <cudf_test/base_fixture.hpp>
 
-#include <vector>
+#include <cudf/utilities/span.hpp>
```

Comment:
I don't see any span is used in this file?

Reply:
Thanks. I had added spans but then removed.